### ISSUE TITLE
[Bug]: `transform-origin` is misattributed as unknown property

### ISSUE THREAD

**Issue by fweth** [jsx-eslint/eslint-plugin-react#3914]
### Is there an existing issue for this?

- [x] I have searched the existing issues and my issue is unique
- [x] My issue appears in the command-line and not only in the text editor

### Description Overview

In SVG elements, like `rect`, `transform-origin` is misattributed as unknown property, but it is a [valid attribute](https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Attribute/transform-origin).

```js
<svg>
  <rect transform-origin="center" />
</svg>
```

I get the following message:

> Unknown property 'transform-origin' foundeslint[react/no-unknown-property](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/no-unknown-property.md)

### eslint-plugin-react version

7.37.4

### eslint version

9.23.0

### node version

v22.14.0

**Comment by fweth** [comment]
Hi, thanks a lot for fixing this! Just one question, do I understand correctly that `transform-origin` has been only added to the `rect` element? That was just an example in my ticket, but the property is [supported](https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Attribute/transform-origin) on all SVG elements!

**Comment by ljharb** [comment]
Yes, i only added it to `rect`. I’ll reopen until all of them are covered.
